packages/color-area/src/ColorArea.ts

```diff
@@ -498,7 +504,7 @@ export class ColorArea extends SpectrumElement {
                     type="range"
                     class="slider"
                     name="x"
-                    aria-label=${this.label}
+                    aria-label=${this.label ?? this.labelX}
```

Comment:
I may not fully understand the `??` usages here, but should these resolve the other way so that users that accidentally make a PR like the following they get the more custom values?
```
<sp-color-area
    label="Both things"
+  label-x="One thing"
+  label-y="Other thing"
></sp-color-area>
```

Reply:
Also, we should make a note somewhere, an issue or code comment or something else you  might prefer, to deprecate support for this in the future, hopefully when we got 1.0 or make a specifically breaking `0.x.0` release.

Reply:
That *is* how I would do it if this were a new component (is it new enough that we could consider it a new component?).

The reason I have `label` overriding `label-x` / `label-y` is so that it can be totally backwards-compatible. If someone today is using this and specifying `label`, their `label` value will not break if they upgrade to this version. Anyone who isn't specifying a custom `label` will get the new, more-accessible behavior.

Reply:
OK, I see now, it's the fact that you moved default values into the `label-*` properties. Mystery of the `??` solved.

We could technically put some sort of getter/setter or change tracking to _unset_ `label` when `label-*` values are set from the outside to ensure that new values are preferred, but that's likely too much extra paths to protect accidentally setting all three properties.

Let's still comment/issue create, but I'm onboard with this!